README.md

```diff
@@ -124,6 +124,18 @@ browser.mouse
 browser.quit
 ```
 
+Evaluate JavaScript to modify things before a page load:
+
+```ruby
+javascript = <<~JS
+  Object.defineProperty(navigator, 'languages', {
+    get: function() { return ['tlh']; }
+  });
+JS
+
+Browser.new(evaluate_on_new_document: javascript)
```

Comment:
I think we already have `:extensions` we can say in the description of this newly added method that they are related, no need to introduce new option.

Reply:
I'm not quite sure how you meant this exactly but I changed it how I understood it.
Can you check again please?